### PR TITLE
Update shikigami weapon

### PR DESCRIPTION
**Pull request by cmastro24**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Shikigami weapon will not go out of invisible on Agro.  I added a status to set it to Normal to force him out of invisible.

## Steps to test these changes

Go to [Ro'Maeve].  
Track Shikigami weapon.  
Cast a spell like Refresh to force him to agro you.  
Validate you are able to engage him.

### CODE REVIEW

File: scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua [hunk 1]
```diff
@@ -49,6 +49,7 @@ entity.onMobSpawn = function(mob)
 end
 
 entity.onMobEngaged = function(mob, target)
+    mob:setStatus(xi.status.NORMAL)
     mob:setStatus(xi.status.UPDATE)
```
Comment (TeoTwawki) [inline scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: Double setStatus for this shouldn't be needed and indicates something happening incorrectly core side if this affects anything at all.

Comment (TeoTwawki) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: Teo_2023 09 06_180904

So I am not seeing the bug on stock LSB

Comment (cmastro24) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: Is he able to be engaged?  I think that's what the original bug report to us said.  I'll retest to see if that's the case.

Comment (TeoTwawki) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: We engaged and killed it. my Char was the bystanding observer.

Everyone was able to and saw.

(moz initially had it, and you can see testaru engaged after in the screenshot with it unclaimed)

Comment (Mortalelite) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: I can say, that I have been able to cast around it and was able to kill it with each pop.

Comment (cmastro24) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: Thank you, I will close the PR.

Comment (cmastro24) [reply scripts/zones/RoMaeve/mobs/Shikigami_Weapon.lua:53]: This may be fork specific

**Comment by cmastro24** [comment]
Closing pull request

**Pull request closed without merging**